consensus/reactor.go

```diff
@@ -351,10 +351,6 @@ func (conR *ConsensusReactor) Receive(chID byte, src p2p.Peer, msgBytes []byte)
 	default:
 		conR.Logger.Error(fmt.Sprintf("Unknown chId %X", chID))
 	}
-
-	if err != nil {
```

Comment:
why remove?

Reply:
`nilness: impossible condition: nil != nil (govet)
        if err != nil {`

this is the linting error I was receiving, when looking through the function everywhere err is being set it is being checked as well